src/xdocs-examples/resources-noncompilable/com/puppycrawl/tools/checkstyle/checks/imports/importorder/Example1.java

```diff
@@ -5,19 +5,23 @@
   </module>
 </module>
 */
+//non-compiled with javac: Compilable with Java11
+package com.puppycrawl.tools.checkstyle.checks.imports.importorder;
 
 // xdoc section -- start
 import java.io.IOException;
 import java.net.URL;
 
-import java.io.IOException; // violation, extra separation before import
-                            // and wrong order, comes before 'java.net.URL'.
+import java.io.IOException;
+// 2 violations above:
+//  'Extra separation in import group before 'java.io.IOException''
+//  'Wrong order for 'java.io.IOException' import.'
 import javax.net.ssl.TrustManager; // violation, extra separation due to above comment
 import javax.swing.JComponent;
 import org.apache.http.conn.ClientConnectionManager; // OK
 import java.util.Set; // violation, wrong order, 'java' should not come after 'org' imports
 import com.neurologic.http.HttpClient; // violation, wrong order, 'com' imports comes at top
 import com.neurologic.http.impl.ApacheHttpClient; // OK
 
-public class SomeClass { }
+public class Example1 { }
 // xdoc section -- end
```

Comment:
Please put `// xdoc section -- end` above class.
Users don't need class definition.

Reply:
Apply to all examples

Reply:
> Apply to all examples

Done